src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = cn({
+              'is-active': selectedTab.id === tab.id,
+            });
```

Comment:
```suggestion
            const isTabSelected = selectedTab.id === tab.id;
```
The variable is called `isTabSelected` but for some reason it contains a call of classname.